liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateViewGenerator.java

```diff
@@ -4,24 +4,24 @@
 import liquibase.database.Database;
 import liquibase.database.core.*;
 import liquibase.exception.DatabaseException;
-import liquibase.structure.core.Relation;
 import liquibase.exception.ValidationErrors;
 import liquibase.sql.Sql;
 import liquibase.sql.UnparsedSql;
 import liquibase.sqlgenerator.SqlGeneratorChain;
 import liquibase.statement.core.CreateViewStatement;
+import liquibase.structure.core.Relation;
 import liquibase.structure.core.View;
 import liquibase.util.SqlParser;
 import liquibase.util.StringClauses;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
 import static liquibase.sqlgenerator.core.CreateProcedureGenerator.splitSetStatementsOutForMssql;
 
 public class CreateViewGenerator extends AbstractSqlGenerator<CreateViewStatement> {
 
+    private static final Set<String> ORACLE_STATEMENT_DELIMITERS = Set.of(";", "/");
```

Comment:
The last piece, just to confirm - did Amy agreed to remove the slash too from the scripts?

Reply:
![image](https://github.com/Datical/liquibase/assets/126700736/b1ba36db-79e1-426f-9788-758ccb5c5766)
Also, you've already done a particular fix of this problem some time ago in DaticalJDBCExecutor via regexp. It is just a complete fix for Oracle+View case.